tests/module/test_types.py

```diff
@@ -146,3 +146,34 @@ def test_from_json_float(self):
         assert target.ra.degree == 10
         assert isinstance(target.dec, Angle)
         assert target.dec.degree == 20
+
+
+class MjdObject(BaseModel):
+    time: aeonlib.types.TimeMJD
+
+
+class TestTimeMJD:
+    def test_from_jd_float(self):
+        obj = MjdObject(time=2460826.5)
+        assert isinstance(obj.time, Time)
+        assert obj.time.to_datetime() == datetime(2025, 5, 31)
+
+    def test_from_mjd_float(self):
+        obj = MjdObject(time=60826.0)
+        assert isinstance(obj.time, Time)
+        assert obj.time.to_datetime() == datetime(2025, 5, 31)
+
```

Comment:
The above tests should test the `.scale` value as well to ensure they are in the correct `'tt'` (TT) timescale. This (new) test fails both of the last 2 `asserts` for example:
```python

    def test_from_datetime(self):
        obj = MjdObject(time=datetime(2025, 5, 5))
        assert isinstance(obj.time, Time)
        assert obj.time.scale == 'tt'
        assert obj.time.tt.mjd == 60800.0
```

Reply:
Good catch, fixed in 2d7feef07cc50f081019699491e628a31bd86396